tiled/adapters/hdf5.py

```diff
@@ -150,3 +151,6 @@ def _items_slice(self, start, stop, direction):
         if direction < 0:
             items = reversed(items)
         return items[start:stop]
+
+    def inlined_contents_enabled(self, depth):
+        return depth < INLINED_DEPTH
```

Comment:
Should this be `<=`? If the depth limit is 7 and this node is 7 deep, enable inlining?

Reply:
Good point.